### PR TITLE
Ensure xml content type for twiml add logging

### PR DESCRIPTION
**Pull request by joliveros**
- Update readme.
- fix database connection string parsing.
- Add [logback-access logging](https://logback.qos.ch/recipes/captureHttp.html)

### CODE REVIEW

File: src/main/java/com/twilio/survey/util/AppSetup.java [hunk 1]
```diff
@@ -2,79 +2,89 @@
...
+		return env.get("DATABASE_URL");
+	}
+
+	/**
+	 * Fetches the environment variable TWILIO_DISABLE_DB_SSL. This needs to be
+	 * set to true if the database doesn't have SSL enabled
+	 */
+	public boolean getSslEnabledDb() {
+		String ssl = env.get("TWILIO_DISABLE_DB_SSL");
+		if (ssl != null && ssl.compareToIgnoreCase("true") == 0) {
```
Comment (hortega) [inline src/main/java/com/twilio/survey/util/AppSetup.java]: This whole method could be replaced with `return !Boolean.parseBoolean(env.get("TWILIO_DISABLE_DB_SSL"));`

File: src/test/java/com/twilio/survey/controllers/QuestionControllerTest.java [hunk 2]
```diff
@@ -62,7 +63,7 @@ public void getNoQuestionSMSTest() throws Exception {
 
         assertThat(response, containsString(
                 "<Message><Body>We are sorry, there are no more questions available for this survey. Good bye" +
-                        ".</Body></Message>"));
+                        ".</Body></Message>"));
```
Comment (well1791) [inline src/test/java/com/twilio/survey/controllers/QuestionControllerTest.java]: @joliveros extra space?

File: src/main/java/com/twilio/survey/util/AppSetup.java [hunk 3]
```diff
@@ -2,79 +2,74 @@
...
+    if (userInfo.isEmpty()) {
+      String[] credentials = userInfo.split(":");
+      queryStringParams.add(new BasicNameValuePair("username", credentials[0]));
 
-        String dBUrl =
-                "jdbc:postgresql://" + dbUri.getHost() + ":" + dbUri.getPort() + dbUri.getPath() +
-                        "?user=" + username + "&password=" + password;
-        if (getSslEnabledDb()) {
-            dBUrl += "&ssl=true&sslfactory=org.postgresql.ssl.NonValidatingFactory";
+      String password = credentials[1];
```
Comment (well1791) [inline src/main/java/com/twilio/survey/util/AppSetup.java]: @joliveros it's possible to do something like
```java
String password = credentials[1].isEmpty() ? "" : credentials[1];
```

File: src/main/java/com/twilio/survey/util/AppSetup.java [hunk 4]
```diff
@@ -2,79 +2,74 @@
...
-        if (userInfo.length > 1) {
-            password = dbUri.getUserInfo().split(":")[1];
-        } else {
-            password = "";
-        }
+  /**
+   * Formats the database url to match JDB format
+   *
+   * @param url
+   *            receives the url string from the environment variable
```
Comment (well1791) [inline src/main/java/com/twilio/survey/util/AppSetup.java]: @joliveros is there a reason to move _receives ..._ to a new line?

**Comment by hortega** [comment]
LGTM!

**Pull request merged**